youtube_dl/extractor/iqm2.py

```diff
@@ -0,0 +1,113 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..compat import compat_urlparse
+from .jwplatform import JWPlatformBaseIE
+from ..utils import js_to_json
+
+# IQM2 aka Accela is a municipal meeting management platform that
+# (among other things) stores livestreamed video from municipal
+# meetings.  In some cases (e.g. cambridgema.iqm2.com), after a hefty
+# (several-hour) processing time, that video is available in easily
+# downloadable form from their web portal, but prior to that, the
+# video can only be watched in realtime through JWPlayer. Other
+# (somervillecityma.iqm2.com) instances don't seem to ever offer a
+# downloadable form. This extractor is designed to download the
+# realtime video without the download links being available. For more
+# info on Accela, see:
+#   http://www.iqm2.com/About/Accela.aspx
+#   http://www.accela.com/
+#   https://github.com/Accela-Inc/leg-man-api-docs
```

Comment:
Done. Sorry, this _really_ goes against the grain for me. I've been trained to write with extneisve comments, and I cannot at all understand why you don't want good comments in youtube-dl. Every time I read another extractor, I wish there was more context explaining what it was and why the choices were made and how it applies and where to look for more information. So it's very difficult for me to omit those things.  Sorry if it feels like I'm not honoring your coding standards, I'm really trying but it is hard for me.